build.sbt

```diff
@@ -406,7 +406,6 @@ lazy val compiler = configureAsSubproject(project)
         (products in Compile in packageBin in LocalProject("interactive")).value ++
         (products in Compile in packageBin in LocalProject("scaladoc")).value ++
         (products in Compile in packageBin in LocalProject("repl")).value ++
-        (products in Compile in packageBin in LocalProject("repl-jline")).value ++
```

Comment:
`packageBin` doesn't prove anything, it was never intended to work.

The REPL tries multiple JLine options and picks the best one:

```
$ scala -nc -Ydebug
Trying to instantiate a InteractiveReader from scala.tools.nsc.interpreter.jline.InteractiveReader
Trying to instantiate a InteractiveReader from scala.tools.nsc.interpreter.jline_embedded.InteractiveReader
All InteractiveReaders tried:
  - scala.tools.nsc.interpreter.jline.InteractiveReader OK
  - scala.tools.nsc.interpreter.jline_embedded.InteractiveReader OK
```

Both `jline` and `jline_embedded` should show up here in a Scala distribution that you build.